data/remnant missions.txt

```diff
@@ -493,5 +558,748 @@ mission "Remnant: Technology Available"
+
+			`	"Don't worry," he sings with a tinge of reproach. "I am a Remnant, not a Korath."`
+				goto more
+
+			label deep
+			`	He clips his harness to one of the overhead lifts and presses a button on the back of his gauntlet, and is promptly whisked up into the air. The lift appears designed to give engineers and techs almost as much freedom as a zero-g environment by suspending them with just enough force to counteract gravity, so they can move around a ship with ease. After a quick once-over he quickly gravitates to the weapon ports and turret hardpoints.`
+
+			label more
+			`	After examining <ship> for quite some time, he calls down to you. "I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at a Plasma Cannon. You nod, and he continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could acquire some for us you would be well compensated."`
+			choice
+				`	"Sure, I would be glad to work out a deal."`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, he agrees to pay you one million credits in exchange for a cargo of two "Plasma Cannons."`
+				accept
+	
+	on complete
+		outfit "Plasma Cannon" -2
+		payment 1500000
+		conversation
+			`As <ship> settles onto the pad closest to the research facility, you can see the weapons specialist checking over a large cannon on a platform beneath a nearby Albatross. When he sees you starting to unload the two plasma cannons he snaps the casing closed and gestures to someone out of sight. Within moments both him and an empty flatbed are there to pick up the weapons.`
+			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
+			`	The weapon tech finishes strapping down the plasma cannons, and looks up to see another Remnant approaching. He gives the new Remnant a respectful dip of his head that almost seems like a bow before turning to you and chanting, "Ah, Captain <last>, have you met Tali? She is the engineering prefect here on Viminal. She asked me to let her know when or if you managed to procure the new tech for me."`
+			`	"Well met, Newcomer among the Remnant," trills the prefect. "Fresh blood amongst us sets songs in motion that have long been in Caesura. I have a few things to finish up right now, but I hope to work with you soon." She gives you a polite nod and heads back the way she came.`
+			
+mission "Remnant: Return the Samples"
```

Comment:
1. Maybe they are. We don't really know, do we? Also, there is mention of having been stored in cryogenic stasis to provide an explanation. Also, it seems reasonable to me that a life form that can naturally travel in space would likely withstand the stress of cryogenics a fair bit better than humans do.

2. Might have been, might not. Stealing the progeny of something ranks pretty high up in terms of crimes. 

Well, in this particular case, I specifically want to redeem the *player* not the Remnant. The Archon is both sentient and telepathic, so it can almost certainly tell the difference. The Remnant will need to do more before they are redeemed, if ever. Personally, if I was stationed in a system to preserve a species of creature, I'd be much more upset by someone stealing a pile of eggs than I would be over a couple killed. Creatures die, that's part of life. 

Honestly, the most likely thing is that the Remnant killed a few Sprites, then stole said sprite's entire clutch of eggs. So not only did they kill the Sprites, they removed an entire generation from that particular set of sprite's gene pool. That could potentially mess things up a lot if there aren't very many of them around. (and we really have no idea how many of them there are)